apps/condo/domains/organization/schema/ResetOrganizationService.js

```diff
@@ -88,18 +89,12 @@ const ResetOrganizationService = new GQLCustomSchema('ResetOrganizationService',
                     await PropertyIdOnly.softDelete(context, property.id, DV_SENDER)
                 }
 
-                const employees = await loadListByChunks({
-                    context,
-                    list: OrganizationEmployee,
-                    chunkSize: 20,
-                    limit: 100000,
-                    where: {
-                        deletedAt: null,
-                        organization: { id: organizationId },
-                    },
+                const employees = await find('OrganizationEmployee', {
```

Comment:
it's nice `loadListByChunks` replacement. 🥳
We can find others places and replace it like so!